client/src/components/QA/QACardQuestions.jsx

```diff
@@ -1,38 +1,75 @@
-import React, { useEffect, useState } from 'react';
-import Modal from 'react-modal';
-import axios from 'axios';
-import QuestionsAndAnswers from './QuestionsAndAnswers.jsx';
+import React, { useState, useEffect } from 'react';
 import QACardAnswers from './QACardAnswers.jsx';
 import QAAddAnswerModal from './QAAddAnswerModal.jsx';
+import QAAddQuestionModal from './QAAddQuestionModal.jsx';
 
-const QACardQuestions = ({ questions, productName }) => {
+const QACardQuestions = ({ questionsInfo, productName }) => {
 
   const [modalIsOpenAdd, setModalIsOpenAdd] = useState(false);
+
   // NOTE - keeping for refactor later on based on server changes for this path
   // // answers array for current product
   // const [answers, setAnswers] = useState([]);
 
+  const [word, setWord] = useState('');
+  const [length, setLength] = useState(4);
+
   // useEffect(() => {
-  //   axios.get('/qa/questions/:question_id/answers')
-  //     .then(response => {
-  //       setAnswers(response.data);
-  //     })
-  //     .catch(err => err);
+  // axios.get('/qa/questions/:question_id/answers')
+  //   .then(response => {
+  //     setAnswers(response.data);
+  //   })
+  //   .catch(err => err);
   // }, []);
 
+  const increaseCount = () => {
+    event.preventDefault();
+    setLength(length + 2);
+  };
+
+  const filterQuestions = () => {
+    if (event.target.value.length < 3) {
+      setWord('');
+    } else {
+      setWord(event.target.value);
+    }
+  };
+
   return (
     <div data-testid="qa-questions">
-      <div>{questions.map((question, index) =>
-        <div className="qa-card-sample" key={index}>
-          <b><div className="qa-div">Q: {question.question_body}
-            <div className="qa-helpfulness-right" data-testid="qa-helpfulness-right">Helpful? <a href='' className='right-spacing'>Yes ({question.question_helpfulness}) </a>
-              <QAAddAnswerModal question={question} productName={productName} />
+      <form data-testid="search" id="search">
+        <input className="qa-searchbar" data-testid="qa-searchbar" placeholder="HAVE A QUESTION? SEARCH FOR ANSWERS..."
+          onChange={() => {
+            filterQuestions();
+            setLength(4);
+          }}
+        ></input>
+        {/* NOTE - Search button will be replaced with icon later on */}
```

Comment:
If you make an icon/button modular, we could reuse in several places on the site.